### PR TITLE
New Game : Bubble'z popper

### LINKED ISSUE #4212: [New game]: Bubble'z Popper
### 🎮 Game Request

Pop the Bubbles

A simple bubble-popping game made using HTML, CSS, and Javascript.
By moving the cursor onto the bubbles they blast.If you miss the 10 chances the game will end.

https://github.com/kunjgit/GameZone/assets/125206920/af0a669d-1e5e-40b5-bdd4-e32745c0a399

 

### Point down the features

-Points increase based on number of bubbles blasted.
-Sounds are added for good user experiences.
-Works on mobile too.the 

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Chandu6702**
## PR Description 📜

I added a simple bubble-popping game using HTML, CSS, and Javascript.
By moving the cursor onto the bubbles they blast. If you miss the 10 chances the game will end.

Fixes #4212 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸
Screenshot 2024-06-04 133640

https://github.com/kunjgit/GameZone/assets/125206920/64271834-48f4-41fb-ba59-dadf44ea453e

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Chandu6702 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by Chandu6702** [comment]
@kunjgit  please review and merge this PR

**Comment by Chandu6702** [comment]
@kunjgit please review and merge this PR

**Comment by Chandu6702** [comment]
@kunjgit please review and merge this PR

**Comment by Chandu6702** [comment]
@kunjgit please review and merge this PR

**Comment by Chandu6702** [comment]
@kunjgit please review and merge this PR

**Comment by Chandu6702** [comment]
@kunjgit please review and merge this PR

**Comment by Chandu6702** [comment]
@ishita-43 please review this PR

**Comment by Chandu6702** [comment]
@kunjgit please review and merge this pr

**Comment by Chandu6702** [comment]
@ishita-43 please review this PR

**Comment by Chandu6702** [comment]
@ishita-43 and @kunjgit please review this PR
It's been 3 days since I created this PR,
Other PRs which are created later have been merged
I don't know what's the problem can you please review this PR and let me know if I need to change anything.

File: README.md [hunk 1]
```diff
@@ -335,6 +335,14 @@ This repository also provides one such platforms where contributers come over an
 
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light) |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)
```
Comment (ishita-43) [inline README.md:338]: remove duplicate entry

File: README.md [hunk 2]
```diff
@@ -335,6 +335,14 @@ This repository also provides one such platforms where contributers come over an
 
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light) |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
+| [Bubble'z Popper](https://github.com/Chandu6702/GameZone/tree/Bubble'z-Popper/Games/Bubble'z%20Popper
+|[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)
```
Comment (ishita-43) [inline README.md:341]: remove

File: README.md [hunk 3]
```diff
@@ -335,6 +335,14 @@ This repository also provides one such platforms where contributers come over an
 
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light) |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
+| [Bubble'z Popper](https://github.com/Chandu6702/GameZone/tree/Bubble'z-Popper/Games/Bubble'z%20Popper
+|[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)                  
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)          |
```
Comment (ishita-43) [inline README.md:342]: remove

File: README.md [hunk 4]
```diff
@@ -335,6 +335,14 @@ This repository also provides one such platforms where contributers come over an
 
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light) |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
+| [Bubble'z Popper](https://github.com/Chandu6702/GameZone/tree/Bubble'z-Popper/Games/Bubble'z%20Popper
+|[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)                  
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)          |        
+|[Dsa_quiz_game](https://github.com/kunjgit/GameZone/tree/main/Games/Dsa_quiz_game)   |              
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)   |
```
Comment (ishita-43) [inline README.md:344]: remove

File: README.md [hunk 5]
```diff
@@ -335,6 +335,14 @@ This repository also provides one such platforms where contributers come over an
...
+| [Gravity_Simulation_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Gravity_Simulation_Game)   |
```
Comment (ishita-43) [inline README.md:345]: remove

File: README.md [hunk 6]
```diff
@@ -335,6 +335,14 @@ This repository also provides one such platforms where contributers come over an
 
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light) |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
+| [Bubble'z Popper](https://github.com/Chandu6702/GameZone/tree/Bubble'z-Popper/Games/Bubble'z%20Popper
+|[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)                  
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)          |        
+|[Dsa_quiz_game](https://github.com/kunjgit/GameZone/tree/main/Games/Dsa_quiz_game)   |
```
Comment (ishita-43) [inline README.md:343]: remove

File: README.md [hunk 7]
```diff
@@ -335,6 +335,14 @@ This repository also provides one such platforms where contributers come over an
 
 |[Turn_on_the_light](https://github.com/kunjgit/GameZone/tree/main/Games/Turn_on_the_light) |
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
+| [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
+| [Bubble'z Popper](https://github.com/Chandu6702/GameZone/tree/Bubble'z-Popper/Games/Bubble'z%20Popper
+|[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
```
Comment (ishita-43) [inline README.md:340]: remove

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by Chandu6702** [comment]
@ishita-43 I have resolved the issues and updated the README.md file,please review this PR

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
@Chandu6702 resolve merge conflicts

**Comment by Chandu6702** [comment]
@kunjgit I resolved the conflicts please review and merge this PR

**Comment by kunjgit** [comment]
Thank you  @Chandu6702 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**